README.md

```diff
@@ -19,23 +19,57 @@ Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
 ### Heroku
 
-First, be sure `MIX_ENV` is set to `heroku` during compilation
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```

Comment:
for heroku "SENDGRID_API_KEY" won't get created automatically

Reply:
true, I will try to make the whole heroku setup documentation a bit better in a future PR then
thanks